lib/ukallocbbuddy/Config.uk

```diff
@@ -5,3 +5,15 @@ menuconfig LIBUKALLOCBBUDDY
 	select LIBUKDEBUG
 	select LIBUKALLOC
 
+if LIBUKALLOCBBUDDY
+
+config UKLIBALLOCBUDDY_THRESHOLD
```

Comment:
Please always use `LIBUKALLOCBBUDDY_` prefix for all options.

Reply:
I'd also prefer if this option is optional, so that `ukalloc_bbuddy` works still without physical allocations. Can you add an option to enable this behavior (it would be okay to have it default on if phys memory mgmt is available)? ... or maybe better: Could you provide another initializer function for `bbuddy` that enables phys. allocation (and a given initial virtual memory area) and allows handing over a physical context? `lib/ukboot` could then initialize `bbuddy` differently depending on what is available (only virtual memory, like linuxu (potentially solo5) or phys. memory mgmt is there). For both options, they should only be available if `ukvmem` or `ukfalloc` is available (depending which interface you use).